docs/02_cleos/03_command-reference/net/peers.md

```diff
@@ -1,16 +1,112 @@
+## Command
+```sh
+cleos net peers [OPTIONS]
+```
+
+**Where:**
+* [OPTIONS] = See **Options** in the [**Command Usage**](command-usage) section below.
+
+**Note:** The arguments and options enclosed in square brackets are optional.
+
 ## Description
-status of all existing peers
+Returns a list with the status of all peer connections. This command allows a node operator to check the status of a node's peer connections.
+
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos net peers` command:
 
-**Command**
+### Positionals
+* `host` _TEXT_ REQUIRED - The hostname:port to disconnect from.
```

Comment:
Full stop at end ... template indicate this list doesn't have them

Reply:
Removed.